test/cypress/support/command/command.js

```diff
@@ -0,0 +1,106 @@
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+
+const TEST_ID_KEY = 'data-testid';
+
+Cypress.Commands.add('getByTestId', (selector, ...args) => cy.get(`[${TEST_ID_KEY}=${selector}]`, ...args));
```

Comment:
instead of creating from scratch new commands like `getByTestId` and the `getByText` ones, better relying on `cypress-testing-library` which provides them out of the box and makes devs using the same function signatures we already use in `react-testing-library`.